docs/faq.md

```diff
@@ -72,15 +72,17 @@ You've added new worker groups. Deleting worker groups from earlier in the list
 
 The safest and easiest option is to set `asg_min_size` and `asg_max_size` to 0 on the worker groups to "remove".
 
-## Why does changing the worker group's desired count not do anything?
+## Why does changing the node or worker group's desired count not do anything?
 
-The module is configured to ignore this value. Unfortunately Terraform does not support variables within the `lifecycle` block.
+The module is configured to ignore this value. Unfortunately, Terraform does not support variables within the `lifecycle` block.
 
-The setting is ignored to allow the cluster autoscaler to work correctly and so that terraform apply does not accidentally remove running workers.
+The setting is ignored to allow the cluster autoscaler to work correctly so that terraform apply does not accidentally remove running workers.
```

Comment:
```suggestion
The setting is ignored to allow the cluster autoscaler to work correctly so that `terraform apply` does not accidentally remove running workers.
```